errors/format.go

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2017 The Things Network
+// Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+
+package errors
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/gotnospirit/messageformat"
```

Comment:
The whole idea behind using an open standard like ICU is that we don't have to depend on a certain implementation of it. Sure the `gotnospirit` package is not widely used or maintained, but it is feature-complete and just works.

ICU is a really simple format, so we could maintain this ourselves if we wanted to (I even had a jab at implementing ICU in go, it's easy).

The go template syntax is not meant for localising messages, and does not have support for plurality, gender, ... And it's not a standard, so implementing this in other languages will be a lot  harder.

On top of that, this is also the format the Console uses as well.